_extras/snippets_library/UCL_sge/13/long_job.snip

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
```

Comment:
`#!/bin/bash -l` 

We try to emphasise that they always leave in the `-l` for a login shell, as otherwise the job won't get their normal environment, including modules. For this example it would work anyway but we want to avoid people doing it.